tests/unity/example_test/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Kconfig - Config options for logger sample app
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+mainmenu "Example unity test"
+
+menu "UUT module configuration"
```

Comment:
I'm not sure a user opening `menuconfig` for this example knows what UUT means.
At least I don't !

Reply:
Fixed, replaced with full name (unit under test)